working/2509-private-named-parameters/feature-specification.md

```diff
@@ -459,18 +552,33 @@ normative*, but is merely suggestions and ideas for the implementation teams.
 They may wish to implement all, some, or none of this, and will likely have
 further ideas for additional warnings, lints, and quick fixes.
 
+### Error messages
+
+Compile errors and runtime exceptions (from things like invalid dynamic calls)
+may sometimes show a function's signature to the user or otherwise refer to a
+formal parameter. When tools generate these error strings, they should use the
+access name of each parameter if the error relates to a *use* of the parameter
+list. *For example, an error related to calling a function with an incorrect
+parameter type.*
+
+Errors that refer to accessing the parameter from within the function or
```

Comment:
... accessing the parameter variable ...

(to make a distinction between the parameter and its introduced variable.)